### ISSUE TITLE
Disabling "mention" causes the "upload-image" button to disappear

### ISSUE THREAD

**Issue by aaugustin** [agusmakmun/django-markdown-editor#163]
## Details

- OS (Operating System) version: macOS 11.4
- Browser and browser version: Firefox 89
- Django version: 3.2.5
- Martor version & theme: 1.6.3; the bug happens with both themes

### Steps to reproduce

```
MARTOR_ENABLE_CONFIGS = {
    'imgur': 'true',
    # if I comment out this config (I don't need the feature), the image upload button disappears!
    'mention': 'true',
    'jquery': 'true',  # required for the admin
}

MARTOR_TOOLBAR_BUTTONS = [
    'image-upload',
    'toggle-maximize',
]
```

Here's the button that disappears when I disable the mention feature:

<img width="659" alt="Screen Shot 2021-07-14 at 15 08 16" src="https://user-images.githubusercontent.com/788910/125627464-e20b1da7-6842-487c-a502-e2e4442a271e.png">

**Comment by aaugustin** [comment]
This appears to be a known behavior: the instructions in the wiki for customizing image uploads explicitly enable the "mention" feature. Still, it's surprising.

**Comment by agusmakmun** [comment]
good catch @aaugustin, its now fixed on v1.6.4, please upgrade your martor:

```
pip install martor --upgrade
```

**Comment by aaugustin** [comment]
Whoa — that was fast! Thank you very much!
